pubspec.yaml

```diff
@@ -1,10 +1,10 @@
 name: volume_controller
 description: A Flutter volume plugin for ios and android control system volume.
-version: 2.0.3
+version: 2.0.7
 homepage: https://github.com/kurenai7968/volume_controller
 
 environment:
-  sdk: ">=2.12.0 <3.0.0"
+  sdk: ">=2.12.0 <4.0.0"
```

Comment:
If we are updating to Dart 3, looks important to define the minimum SDK to 3.0 since it will not work on the new features, isn't it?